src/com/company/Main.java

```diff
@@ -8,18 +8,40 @@
 
 
 public class Main {
-    public static void main(String[] args){
+    public static void main(String[] args) {
+
+        String txtFilePath = args[0];
```

Comment:
ArrayIndexOutOfBounds exception will be thrown here if user does not pass any argument. Please check.

Reply:
resolved that exception and updated in the recently committed code